forms-flow-documents/requirements/prod.txt

```diff
@@ -16,4 +16,5 @@ markupsafe
 PyJWT
 selenium==3.141.0
 selenium-wire==3.0.0
+nested-lookup==0.2.25
```

Comment:
can we remove the version from prod.txt, so that `make setup` would always keep it up-to-date on requirements.txt ?

Reply:
👍